README.md

```diff
@@ -0,0 +1,98 @@
+![Github CI Status](https://github.com/oasis-roles/index_href/workflows/Role%20CI/badge.svg)
+
+index_href
+===========
+
+Provides a lookup plugin that can parse links in a web directory index
+to get the URLs for a specific file that can then be used in later tasks.
+
+Only URLs contained in anchor (the `a` tag) href properties are searched.
+
+This plugin does not access the internet. It is intended to be used with
+existing ansible mechanisms where possible. To get HTML content to parse
+with this plugin, the builtin `url` lookup plugin or the builtin `uri`
+module is recommended. To download hrefs returned by this plugin, the
+builtin `uri` module is recommended.
+
+Requirements
+------------
+
+Ansible 2.5 or higher
```

Comment:
![But why?](https://i.imgur.com/Br00TCn.gif)

The only thing stopping me from dropping this to ansible 1.9 is the recommended use of `query`. Is there some reason we should be going out of our way to disallow usage in older ansible installs?

Reply:
We discussed this elsewhere.  Basically, if our strategy is to just bump the minimum version if a problem is discovered with an older version rather than having to make it backwards-compatible forever, I'm cool with that.